modules/toolbox/framework_tools.py

```diff
@@ -21,3 +20,44 @@ def get_train_test(path, filename, separator):
     # Selecting training and test sets
     return cross_validation.train_test_split(x, y, test_size = 0.2)
 
+
+# works for CSVs, not images. Might want to pass in the target column instead of the file
+def get_prediction_type(path, filename, separator):
+    numpy_data = get_data(path, filename, separator)
+
+    y = numpy_data[:, numpy_data.shape[1] - 1]
+
+    classification = is_classification(y)
+    regression = is_regression(y)
+
+    if not (regression or classification):
+        return "invalid"
+    elif classification:
+        return "classification"
+    else:
+        return "regression"
+
+
+def get_data(path, filename, separator):
+    pandas_data = pd.read_csv(path + filename, sep=separator)
+    return pd.DataFrame.as_matrix(pandas_data)
+
+
+# not positive that this is a good enough check. TODO: Test thoroughly
```

Comment:
@asclines So testing does exist, but the classification recognizer isn't working